examples/chart/teleport/README.md

```diff
@@ -46,16 +46,17 @@ $ helm install --name teleport ./
 Teleport proxy generates a TLS key and a cert of its own by default.
 In order to instruct the proxy to use the TLS assets brought by you, prepare the following files:
 
-- Your CA cert named `ca.pem`
+- Your CA cert named `ca.pem`  (optional depending on values.yaml setting)
 - Your proxy server cert named `proxy-server.pem`
 - Your proxy server key named `proxy-server-key.pem`
 
 Then run:
 
 ```
 $ kubectl create secret tls tls-web --cert=proxy-server.pem --key=proxy-server-key.pem
+# ca.pem is not required by default.  Run this command if you have your own Certificate Authority (CA)
 $ kubectl create configmap ca-certs --from-file=ca.pem
```

Comment:
If it's not required, why do we have the `ca.pem` referenced below?

Reply:
Please review the updated comments.  This is included so you can optionally run this instruction if you have a tls ca

Reply:
PTAL @benarent